helm/values.yaml

```
@@ -110,74 +167,56 @@ config:
-    RULES_FILE: ./rules.json
-    ENABLE_TEST_FEATURES: true
-
-  # set service type to NodePort for local testing on Minikube
-  # set to ClusterIP for kubernetes cluster deployments
-  service:
-    type: NodePort
-    ports:
-      - name: inboundapi
-        targetPort: inboundapi
-        port: 4000
-        protocol: TCP
-      - name: reportapi
-        targetPort: reportapi
-        port: 3002
-        protocol: TCP
-      - name: testapi
-        targetPort: testapi
-        port: 3003
-        protocol: TCP
+    env:
+      ##### Section for simulator backend container #####
+      OUTBOUND_ENDPOINT: "http://localhost:4001" # within the pod
+
+      # Enable mutual TLS authentication. Useful when the simulator is not running in a managed
+      # environment, i.e. when you're running it locally against your own implementation.
+      MUTUAL_TLS_ENABLED: "false"
+
+      # Enable server-only TLS; i.e. serve on HTTPS instead of HTTP.
+      HTTPS_ENABLED: "false"
+
+      # Location of certs and key required for TLS
+      CA_CERT_PATH: ./secrets/cacert.pem
+      SERVER_CERT_PATH: ./secrets/servercert.pem
+      SERVER_KEY_PATH: ./secrets/serverkey.pem
+
+      # The number of space characters by which to indent pretty-printed logs. If set to zero, log events
+      # will each be printed on a single line.
+      LOG_INDENT: "0"
+
+      # The name of the sqlite log file. This probably doesn't matter much to the user, except that
+      # setting :memory: will use an in-memory sqlite db, which will be faster and not consume disk
+      # space. However, it will also mean that the logs will be lost once the container is stopped.
+      SQLITE_LOG_FILE: ./log.sqlite
+
+      # The DFSPID of this simulator. The simulator will accept any requests routed to
+      # FSPIOP-Destination: $SCHEME_NAME. Other requests will be rejected.
+      SCHEME_NAME: golden
+      DFSP_ID: golden
+
+      # The name of the sqlite model database. If you would like to start the simulator with preloaded
+      # state you can use a preexisting file. If running in a container, you can mount a sqlite file as a
+      # volume in the container to preserve state between runs.
+      # Use MODEL_DATABASE: :memory: for an ephemeral in-memory database
+      MODEL_DATABASE: ./model.sqlite
+
+      # The simulator can automatically add fees when generating quote responses. Use this
+      # variable to control the fee amounts added. e.g. for a transfer of 100 USD a FEE_MULTIPLIER of 0.1
+      # reuslts in fees of USD 10 being applied to the quote response
+      FEE_MULTIPLIER: "0.05"
 
+      # Specifies the location of a rules file for the simulator backend. Rules can be used to produce
+      # specific simulator behaviours in response to incoming requests that match certain conditions.
+      # e.g. a rule can be used to trigger NDC errors given transfers between certain limits.
+      RULES_FILE: ./rules.json
```

This appears unused. But we should use it, as I believe users will want to configure rules.

---

Actually, I now realise this is the location inside the container where the rules are expected to be. This should probably read more like
```suggestion
      RULES_FILE: /rules.json
```